Tests/Tests/RouterSpec.swift

```diff
@@ -279,6 +361,38 @@ class RouterSpec: QuickSpec {
                         expect(window.findTooltipView()).toEventuallyNot(beNil())
                     }
 
+                    it("will display a tooltip above the campaign view") {
```

Comment:
What does it mean "above the campaign view". Tooltips should be displayed "under" a campaign view. To check that you need to check the index of tooltip view

Reply:
Yes, you are correct. I want to handle the case when the tooltip is shown above the campaign, the campaign should bring to the front. I update here [f60c555](https://github.com/rakutentech/ios-inappmessaging/pull/275/commits/f60c555517519f44c4bd47352520a9940b80058f).